src/components/ui/HostNavigation.jsx

```diff
@@ -0,0 +1,43 @@
+import { NavLink } from "react-router-dom";
+
+const HostNavigation = () => {
+    const activeStyles = {
+        fontWeight: "bold",
+        textDecoration: "underline",
+        color: "#161616",
+    };
+    return (
+        <div className="flex justify-start gap-4 item-center">
+            <NavLink
+                to="/host/dashboard"
```

Comment:
You can use a relative path to link to the dashboard, as it is the index route.
```suggestion
                to="."
```